mina_serialization_macros/proc_macros/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-authors = ["Granola <hello@granola.team>"]
+authors = ["Mina Foundation, Inc."]
```

Comment:
I think we should retain the original authors value. @robinbb ?

Reply:
Updated.

`authors` now reads:

> ChainSafe Systems Inc., Granola Systems Inc.